src/components/PWDKPhoneNumber.tsx

```diff
@@ -0,0 +1,138 @@
+import 'react-phone-input-material-ui/lib/material.css'
+import { TextField } from '@mui/material'
+import { styled } from '@mui/material/styles'
+import { ChangeEvent, ComponentProps, FC, FocusEvent, useMemo } from 'react'
+import PhoneInput from 'react-phone-input-material-ui'
+
+export const phoneRegExp = /^\d{8,}$/ // Minimum: 8 digit number
+
+// -------------------------- types --------------------------
+
+type SizeCustom = {
+  padding: string
+  height: string
+  fontSize: string
+  lineHeight: string
+}
+
+interface ReactPhoneInputProps extends ComponentProps<typeof PhoneInput> {
+  sizeCustom: SizeCustom
+  error: boolean
+}
+
+type PWDKPhoneNumberProps = {
+  size?: string
```

Comment:
```suggestion
  size?: string
```
ini bisa dikasih pilihan kan ya sizenya apa aja yg avaiable